front-end/src/util/blockToTime.ts

```diff
@@ -9,11 +9,11 @@ function secondsToDhm(seconds: number) {
 	seconds = Number(seconds);
 	var d = Math.floor(seconds / (3600*24));
 	var h = Math.floor(seconds % (3600*24) / 3600);
-	var m = Math.floor(seconds % 3600 / 60);
+	var m = seconds < 60 ? Math.ceil(seconds % 3600 / 60) : Math.floor(seconds % 3600 / 60);
```

Comment:
Added this so that for blocks equaling less than 1 min it'll get rounded up to 1min

Reply:
I guess the first expression will always = 1, so I suggest
```suggestion
	var m = seconds < 60 ? 1 : Math.floor(seconds % 3600 / 60);
```